src/index.ts

```diff
@@ -150,6 +151,145 @@ app.get("/api/users/:id", async (req, res) => {
   }
 });
 
+
+/**
+ * @openapi
+ * /api/users: 
+ *   delete:
+ *     summary: Removes user
+ *     responses:
+ *       200:
+ *         description: Successfully removed a user
+ *         content:
+ *           text/plain:
+ *             schema:
+ *               type: string
+ *       400:
+ *         description: Id has wrong data type
+ *         content:
+ *           text/plain:
+ *             schema:
+ *               type: string
+ *       404:
+ *         description: No user with given id
+ *         content:
+ *           text/plain:
+ *             schema:
+ *               type: string
+ *   parameters:
+ *   - name: id
+ *     in: path
+ *     description: Id of the user to be removed 
+ *     required: true
+*/
+
+app.delete("/api/users/:id", async (req, res) => {
+  const { id } = req.params;
+
+  if (typeof id !== "string") {
+    res
+      .status(400)
+      .send("Id has wrong data type");
+  }
+
+  try {
+    const user = await prisma.user.delete({
+      where: {
+        id: Number(id),
+      },
+    });
+
+    res.send(`User with id ${id} was deleted`);
+  } catch (e) {
+    res.status(404).send(`There is no user with id ${id}`);
+
+  }
+});
+
+/**
+ * @openapi
+ * /api/users: 
+ *   put:
+ *     summary: Updates user data 
+ * 
+ *     requestBody:
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             name: user data
+ *             required: true
+ *             properties:
+ *               name:
+ *                 type: string
+ *                 description: User name
+ *                 example: John Doe
+ *               email:
+ *                 type: string
+ *                 description: User email
+ *                 example: test@test.pl
```

Comment:
There is an issue either in this documentation comment or in actual endpoint implementation:
Here, you describe that the endpoint accepts object with fields name & email as request body.
But [below](https://github.com/FRSOURCE/brownbag-workshop-psql-api/pull/3/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R260) you expect request body to contain an object with `data` field that contains `name` and `email`.

You need to go either one way or another 🙂

Reply:
There is an inconsistency between the swagger documentation and the actual implementation of your endpoint. Here you're specifying what will be a data sent to your endpoint. And you specified that the request `body` should contain `email` and `name`. But [in your code](https://github.com/FRSOURCE/brownbag-workshop-psql-api/pull/3/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R263) you're doing `const { data } = req.body;` - so you expect that within `body` there will be a `data` object holding `email` and `name`.

It's not connected to prisma call at all - it's done fine [few lines below](https://github.com/FRSOURCE/brownbag-workshop-psql-api/pull/3/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R278-R283).

We can have a call about this one if you still have some questions!

Reply:
Yup - now it's alright! As long as the docs follow the implementation its cool 😄